compiler/src/iree/compiler/Codegen/Common/Passes.h

```diff
@@ -259,7 +259,8 @@ createTileAndDistributeToWorkgroupsPass(
 
 /// Create an IREE-specific Transform dialect interpreter pass with all
 /// registrations necessary for IREE.
-std::unique_ptr<Pass> createTransformDialectInterpreterPass();
+std::unique_ptr<Pass>
+createTransformDialectInterpreterPass(StringRef entryPoint = "");
```

Comment:
I think I was confused with `entryPoint` . This is the transform dialect `entryPoint` , i.e. `sequence`. Maybe just rename it to use `transformSequence`?